pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Református énekeskönyv (21-es kék és 48-as fekete)
 
 publish_to: 'none'
 
-version: 2.4.0+14
+version: 2.4.1+15
```

Comment:
Ez a rész különösen tetszik, kreatív megoldás!

Reply:
Ahogy ESR is megmondta: RORE. ;)